src/grpc.ts

```diff
@@ -76,17 +73,16 @@ export interface Metadata {
   get: (key: {}) => {};
 }
 
-export type GrpcModule = typeof grpcTypes & {
-  status: {[index: string]: number};
-};
+export type GrpcModule = typeof grpc;
 
 export interface ClientStubOptions {
   servicePath: string;
   port: number;
-  sslCreds?: grpcTypes.ChannelCredentials;
+  // @ts-ignore waiting for grpc-js types
```

Comment:
I don't understand why this is necessary.  Should `sslCreds` just be `any` until we can fix things?

Reply:
Changed to `any`!